gef-ui/server/controllers/amendments/cover-end-date/get-cover-end-date.ts

```diff
@@ -11,6 +11,7 @@ import { convertDateToDayMonthYearInput } from '../helpers/dates.helper.ts';
 
 export type GetCoverEndDateRequest = CustomExpressRequest<{
   params: { dealId: string; facilityId: string; amendmentId: string };
+  query: { change: string };
```

Comment:
```suggestion
  query: { change: boolean };
```

Should this be changed to Boolean please.

Reply:
@abhi-markan I cannot change the change query to a boolean because the query property must satisfy the RequestQuery type constraint, which expects query parameters to be of type string | RequestQuery | string[] | RequestQuery[] | undefined, but I changed it instead to query: { change?: 'true' }; in order to be type of string or undefined.

@Zainzzkk I updated all the files accordingly